Move of lib/options breaks bin/eslint_d and therefore emacs integration

Commit https://github.com/mantoni/eslint_d.js/commit/ee7481526ab48f0328e02681cda6060cd5ff2085 moves `lib/options` which `bin/eslint_d` still requires when called with `--help` flag.

```
Error: Cannot find module '../lib/options'
Require stack:
- /nix/store/58hm5h0asmqv0myv5r4n9rvqf421myy9-node_eslint_d-10.0.0/lib/node_modules/eslint_d/bin/eslint_d.js
Require stack:
- /nix/store/58hm5h0asmqv0myv5r4n9rvqf421myy9-node_eslint_d-10.0.0/lib/node_modules/eslint_d/bin/eslint_d.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.external_module_.Module._resolveFilename (/ws/bitsandpretzels-livestream/.pnp.js:31823:54)
    at Module._load (internal/modules/cjs/loader.js:725:27)
    at Function.external_module_.Module._load (/ws/bitsandpretzels-livestream/.pnp.js:31679:36)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/nix/store/58hm5h0asmqv0myv5r4n9rvqf421myy9-node_eslint_d-10.0.0/lib/node_modules/eslint_d/bin/eslint_d.js:14:19)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32) {
  code: 'MODULE_NOT_FOUND'
}
```

--- a/bin/eslint_d.js
+++ b/bin/eslint_d.js
@@ -11,7 +11,7 @@ if (cmd === '-v' || cmd === '--version') {
 }
 
 if (cmd === '-h' || cmd === '--help') {
-  const options = require('../lib/options');
+  const options = require('../lib/options-cliengine');
   console.log(options.generateHelp());
   return;
 }

